app/javascript/mastodon/components/dropdown_menu.js

```diff
@@ -16,6 +16,12 @@ class DropdownMenu extends React.PureComponent {
     ariaLabel: "Menu"
   };
 
+  constructor(props) {
+    super(props);
+
+    this.handleClick = this.handleClick.bind(this);
+  }
```

Comment:
[`this.handleClick`](https://github.com/tootsuite/mastodon/pull/3141/files#diff-9cb69e76a6adf193574b8c15b43924fcR27) is using arrow function, it is bind.